src/execution_plan/record.h

```diff
@@ -42,6 +44,10 @@ Record Record_Clone(const Record r);
 // Extends record to accommodate 'len' entries.
 void Record_Extend(Record *r, int len);
 
+// Removes last `count` elements from record.
```

Comment:
maybe rephrase as "remove last k elements from the record" and set the variable name to `k`